helm/values-uat.yaml

```diff
@@ -21,9 +21,14 @@ microservice-chart:
       REDIS_PORT: "6380"
       REDIS_SSL_ENABLED: "true"
       CARTS_MAX_ALLOWED_PAYMENT_NOTICES: "5"
+      ELASTIC_APM_SERVICE_NAME: pagopa-ecommerce-payment-requests-service
+      ELASTIC_APM_APPLICATION_PACKAGES: it.pagopa.ecommerce.payment.requests
+      ELASTIC_APM_SERVER_URL: http://quickstart-apm-http.elastic-system.svc:8200
+      ELASTIC_APM_LOG_SENDING: "true"
     envSecret:
       REDIS_PASSWORD: redis-ecommerce-password
       APPLICATIONINSIGHTS_CONNECTION_STRING: applicationinsights-connection-string
```

Comment:
Remove this secret since we are removing application insight with this PR

Reply:
Thanks :) solved with [0f57261](https://github.com/pagopa/pagopa-ecommerce-payment-requests-service/pull/55/commits/0f57261f50d48c998a7ba029c339f41b17df9764)